.github/actions/rockylinux-build/README.md

```diff
@@ -0,0 +1 @@
+Note: This is generic package which can be used for both Redhat8 and Rockylinux8.
```

Comment:
I think it's more easy if you move everything in the main README instead of creating individual README just for this notice